rules/no-array-callback-reference.js

```diff
@@ -49,14 +49,10 @@ const iteratorMethods = [
 	],
 	[
 		'flatMap',
-		{
-			ignore: [
-				'Boolean',
-			],
-		},
```

Comment:
I made a mistake, this is not reasonable to use in `Array#flatMap()`, it should return an array.